src/Directory.Packages.props

```diff
@@ -0,0 +1,34 @@
+﻿<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
```

Comment:
This props file is just automagically included by some built-in targets?

Reply:
Correct